problemi/boltzmannDistribution/boltzmannDistribution.cpp

```diff
@@ -0,0 +1,144 @@
+#include <TCanvas.h>
+#include <TF1.h>
+#include <TH1D.h>
+#include <TLegend.h>
+
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "boltzmann/boltzmann.h"
+
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include "doctest.h"
+
+void plotDistributionROOT(const std::vector<int> &interactions_list,
+                          int molecules) {
+  TCanvas *c1 = new TCanvas("c1", "Boltzmann Simulations", 1200, 800);
+  c1->Divide(std::ceil(interactions_list.size() / 2.), 2);
+  std::cout << "aa" << std::ceil(interactions_list.size() / 2) << std::endl;
+  int pad{1};
+  for (auto nInteractions : interactions_list) {
+    BoltzmannSimulator sim(molecules);
+    sim.run(nInteractions);
+
+    SimStats st{sim.getStats()};
+    std::cout << "Number of interactions=" << nInteractions
+              << " mean=" << st.mean << " stddev=" << st.stddev << std::endl;
+
+    c1->cd(pad);
+    std::string hname{"hE_" + std::to_string(nInteractions)};
```

Comment:
```suggestion
    std::string const hname{"hE_" + std::to_string(nInteractions)};
```
visto che la funzione e' lunghina, e' meglio mettere const qui